parser-typechecker/src/Unison/Codebase/SqliteCodebase/Migrations/MigrateSchema1To2.hs

```diff
@@ -337,7 +338,8 @@ migrateBranch oldObjectId = fmap (either id id) . runExceptT $ do
 
   newHash <- lift . lift $ Hashing.dbBranchHash newBranch
   newHashId <- lift . lift $ Q.saveBranchHash (coerce Cv.hash1to2 newHash)
-  newObjectId <- lift . lift $ Ops.saveDbBranchUnderHashId v2HashHandle newHashId newBranch
+  stats <- lift . lift $ Ops.namespaceStatsForDbBranch newBranch
+  newObjectId <- lift . lift $ Ops.saveDbBranchUnderHashId v2HashHandle newHashId stats newBranch
```

Comment:
The other option is to just not save stats when we save branches; but it's cheaper to just save them when storing the branch since we've already got everything in memory and deserialized.

This is a small enough tweak to this migration that I don't think it matters much, but open to other perspectives here 🤷🏼‍♂️  A lot of this migration stuff is just kind of annoying haha.